app/src/main/cpp/BrowserWorld.cpp

```diff
@@ -1787,14 +1787,18 @@ BrowserWorld::CreateSkyBox(const std::string& aBasePath, const std::string& aExt
     }
     return;
   }
+#if PICOXR
+  // Pico's OpenXR runtime does not support compressed textures at the moment. Use PNGs in the
+  // meantime.
+  const std::string extension = aExtension.empty() ? ".png" : aExtension;
```

Comment:
Does this work? As far as I can see, `extension` is computed in `BrowserWorld::UpdateEnvironment()` by calling `Skybox::ValidateCustomSkyboxAndFindFileExtension()`, which will probably return `.ktx` (since the KTX assets are bundled in the APK).

Reply:
As mentioned ValidateCustomSkyboxAndFindFileExtension() retrieves the extension of the custom skyboxes, i.e., the ones that the user uploads directly to the device. It has nothing to do with the ones we distribute. This means that unless the user has custom extensions that method will always return the empty string.

Reply:
Ah, I understand now.